README.md

```diff
@@ -1,46 +1,40 @@
-# SiardApi - An API for reading and writing files in the SIARD Format 2.1
+# SiardApi - An API for reading and writing files in the SIARD Format 2.2
 
-This package contains an API for reading and writing files in the
-SIARD Format 2.1
```

Comment:
Add Prerequisites: Java version

Reply:
instead of adding the prerequisite in the readme, I correctly configured the jvm toolchain in build.gradle.kts.

@BeriBoss can you check that this works for you too?